### ISSUE TITLE
Strange sorting order

### ISSUE THREAD

**Issue by nico202** [tuhiproject/tuhi#244]
Hi, I've been using tuhi for some time now, and it keeps working well, thanks! :) Something strange regards the first sync of the month.  Just synced March, and my drawings: they are listed in the following order:
- January 2020
- December 2019
- November 2019
- September 2019
- March 2020
Then I restarted tuhi and they were in the right order.

Thanks, Nicolò

**Comment by whot** [comment]
iirc we use timestamps in UTC, what would be the chances that the timestamps didn't match your current month? i.e. your local time was still Feb or already April when UTC was on March.

**Comment by nico202** [comment]
Thanks, but this should not place it before 2019, should it?

**Comment by whot** [comment]
yeah, true. sounds like there's a bug in the sorting order somewhere. i don't have time right now to debug this though, I'd appreciate it if you could have a look, it'll be in `tuhi/gui/drawingperspective.py`, check the `Flowbox` code first.

**Comment by whot** [comment]
I may have found it.

The code in `_update_drawings` is basically: sort drawings by timestamp, generate a flowbox for each year+month and **add** that flowbox to the layout. Then add the drawing to the flowbox.

So on startup it'll load all drawings from disk and sort those correctly. Then you sync and if it's a new month being synced, a new flow box is being *appended* to the current ones. That's `self.box_all_drawings.add` in line 104. 

So basically, we need this to be smarter and call `pack_start()` or `pack_end()`, depending on the location it needs to be added. This is a `GtkBox` so we're limited in the exact layout but here we only need front or back anyway.
